src/components/views/elements/ImageView.js

```diff
@@ -27,6 +27,12 @@ module.exports = React.createClass({
     displayName: 'ImageView',
 
     propTypes: {
+        mxEvent: React.PropTypes.object,
+        src: React.PropTypes.string.isRequired,
+        link: React.PropTypes.string,
+        width: React.PropTypes.number,
+        height: React.PropTypes.number,
+        size: React.PropTypes.number,
```

Comment:
`size` is a horribly overloaded term in relation to images. `fileSize`? `byteCount`?